garage/np/algos/cma_es.py

```diff
@@ -36,6 +36,7 @@ def __init__(self,
                  discount=0.99,
                  sigma0=1.,
                  **kwargs):
+        self.quick_init(locals())
```

Comment:
Please leave a comment listing the fields that cannot be pickled without this.

Reply:
it seems like you missed `__setstate__` and __getstate__` here

Reply:
It turns out that they don't know manually saving any state.